---
title: Student Paper Presentations
sidebar: mydoc_sidebar
permalink: mydoc_paper_presentations.html 
---

## Overview

Each student has been assigned one journal publication to present in class. The
expected structure of the paper presentations is outlined in this [Slideshow Template](https://docs.google.com/presentation/d/172QKFv_m2CYCwPVqAIZJ5SWzwMUSQYyF6YyXf8IDgiU/edit?usp=sharing). 
A detailed presentation schedule is available on the internal [Course Schedule](https://docs.google.com/spreadsheets/d/1qVvgADKVW6RfkP5wu7tzu3QpStotSotEty7f6AKk8e8/edit#gid=162605681). 
The following lists the assigned papers organized by course project topics.


## Publications organized by course projects

All references in [Paperpile](https://paperpile.com/shared/Ts5JKq)

### ChIP-Seq1 - Motif Prediction    

+ McLeay, Robert C, and Timothy L Bailey. 2010. “Motif Enrichment Analysis: A Unified Framework and an Evaluation on ChIP Data.” BMC Bioinformatics 11: 165. [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20356413)
+ Tompa, M, N Li, T L Bailey, G M Church, B De Moor, E Eskin, A V Favorov, et al. 2005. “Assessing Computational Tools for the Discovery of Transcription Factor Binding Sites.” Nature Biotechnology 23 (1): 137–44. [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/15637633)

### ChiP-Seq2 - Peak Calling  

+ Zhang Y, Liu T, Meyer CA, Eeckhoute J, Johnson DS, Bernstein BE, Nussbaum C, Myers RM, Brown M, Li W, et al (2008) Model-based analysis of ChIP-Seq (MACS). Genome Biol. doi: 10.1186/gb-2008-9-9-r137 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/18798982)
+ Wilbanks EG, Facciotti MT (2010) Evaluation of algorithm performance in ChIP-seq peak detection. PLoS One. doi: 10.1371/journal.pone.0011471 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/20628599)

### RNA-Seq1 - RNA-Seq Aligners 

+ Bray NL, Pimentel H, Melsted P, Pachter L (2016) Near-optimal probabilistic RNA-seq quantification. Nat Biotechnol. doi: 10.1038/nbt.3519 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/27043002)
+ Kim D, Langmead B, Salzberg SL (2015) HISAT: a fast spliced aligner with low memory requirements. Nat Methods 12: 357–360 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/25751142)

### RNA-Seq2 - DEG Methods  

+ Love MI, Huber W, Anders S (2014) Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol 15: 550 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/25516281)
+ Zhou X, Lindsay H, Robinson MD (2014) Robustly detecting differential expression in RNA sequencing data using observation weights. Nucleic Acids Res 42: e91 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/24753412)

### VAR-Seq1 - Functional Prediction

+ DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, Hartl C, Philippakis AA, del Angel G, Rivas MA, Hanna M, et al (2011) A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet 43: 491–498 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/21478889)
+ Shihab HA, Rogers MF, Gough J, Mort M, Cooper DN, Day INM, Gaunt TR, Campbell C (2015) An integrative approach to predicting the functional effects of non-coding and coding sequence variation. Bioinformatics 31: 1536–1543 [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/25583119)


